Content.Shared/Body/Systems/SharedBodySystem.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Content.Shared.Damage;
+using Content.Shared.Standing;
+using Robust.Shared.Containers;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Body.Systems;
+
+[Virtual]
+public abstract partial class SharedBodySystem : EntitySystem
+{
+    private const string BodyContainerId = "BodyContainer";
+
+    [Dependency] private readonly IPrototypeManager _prototypes = default!;
+
+    [Dependency] private readonly SharedContainerSystem _containers = default!;
+    [Dependency] private readonly DamageableSystem _damageable = default!;
+    [Dependency] private readonly StandingStateSystem _standing = default!;
```

Comment:
protected?

Reply:
Done in [`ed73bc5` (#11991)](https://github.com/space-wizards/space-station-14/pull/11991/commits/ed73bc51f677b39de22bf41254dab7c3a5a3ea51), though none of those are used in server and the only things keeping body system from being shared-only are HumanoidSystem.SetLayersVisibility, ApplyMetabolicMultiplierEvent and BeingMicrowavedEvent. Was hoping to be able to move body system to shared but in case more things need to be server-only in the future I suppose it makes sense.